### DISCUSSION TITLE
TaxCollection under RDT?

### DISCUSSION THREAD

**Discussion by RoanOak127** [amzn/selling-partner-api-models#3239] [General]
I help make 3rd party software that connects to the SP-API and need the information in TaxCollection to see if Amazon is handling the VAT for an order in the EU. In Seller Central not every order in the EU has Amazon collecting the VAT so being able to distinguish these orders is important. 

I have the Tax Collection object set up and I'm able to read in Orders and OrderItems fine, but orders that should have a tax collection don't. Do you know if TaxCollection is hidden by the Restricted Data Token? I'm not using it since we don't need the customer's PII.

**Comment by getvicky83** [comment]
i dont think its hidden, as i can get the taxcollection only using accestoken
